src/doc-builders.js

```diff
@@ -46,6 +46,7 @@ function group(contents, opts) {
     type: "group",
     contents: contents,
     break: !!opts.shouldBreak,
+    addedLine: !!opts.addedLine,
```

Comment:
Yeah, wrapping the last function argument is one of the most complex tricks in Prettier and took me some time to understand :) You can just test if the formatting doesn't break in some weird cases.